rover_control_test:
  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50  
  
 # Position Controllers ---------------------------------------
  front_right_joint_position_controller:
    type: effort_controllers/JointPositionController
    joint: front_right_joint
    pid: {p: 100.0, i: 0.01, d: 10.0}
  front_left_joint_position_controller:
    type: effort_controllers/JointPositionController
    joint: front_left_joint
    pid: {p: 100.0, i: 0.01, d: 10.0}
  rear_right_joint_position_controller:
    type: effort_controllers/JointPositionController
    joint: rear_right_joint
    pid: {p: 100.0, i: 0.01, d: 10.0}
  rear_left_joint_position_controller:
    type: effort_controllers/JointPositionController
    joint: rear_left_joint
    pid: {p: 100.0, i: 0.01, d: 10.0}
 # Velocity Controllers ---------------------------------------
  # front_right_joint_position_controller:
  #   type: effort_controllers/JointVelocityController
  #   joint: front_right_joint
  #   pid: {p: 100.0, i: 0.01, d: 10.0}
  # front_left_joint_position_controller:
  #   type: effort_controllers/JointVelocityController
  #   joint: front_left_joint
  #   pid: {p: 100.0, i: 0.01, d: 10.0}
  # rear_right_joint_position_controller:
  #   type: effort_controllers/JointVelocityController
  #   joint: rear_right_joint
  #   pid: {p: 100.0, i: 0.01, d: 10.0}
  # rear_left_joint_position_controller:
  #   type: effort_controllers/JointVelocityController
  #   joint: rear_left_joint
  #   pid: {p: 100.0, i: 0.01, d: 10.0}